Source/diablo.cpp

```diff
@@ -933,15 +924,13 @@ void DiabloParseFlags(int argc, char **argv)
 		} else if (strcasecmp("--save-dir", argv[i]) == 0) {
 			paths::SetPrefPath(argv[++i]);
 		} else if (strcasecmp("--demo", argv[i]) == 0) {
-			demoMode = true;
-			if (!LoadDemoMessages(SDL_atoi(argv[++i]))) {
-				SDL_Log("Unable to load demo file");
-				diablo_quit(1);
-			}
+			demoNumber = SDL_atoi(argv[++i]);

```

Comment:
```suggestion
			demoNumber = SDL_atoi(argv[++i]);
			gbShowIntro = false;
```

Reply:
Added. 😉

offtopic: the intro skipping could be an ini option or? Cause I think this could be something that could be toggled with ingame settings in the future.

Reply:
Maybe, not sure. But i noticed that it's confuses users that the video plays during playback and trying to skip it will case the replay to abort.